app/components/admin/geozones/index_component.html.erb

```diff
@@ -29,3 +29,6 @@
     <% end %>
   </tbody>
 </table>
+<div class="budgets-map">
+  <%= render Shared::MapLocationComponent.new(nil, investments_coordinates: nil, geozones_data: geozones_data) %>
```

Comment:
Do you think it makes sense to only render the map when at least one of the geozones contains GeoJSON data? :thinking:

I'm wondering because there might be people who don't use polygons, and they might see an empty map and wonder what's going on and might expect the "HTML coordinates" to be displayed there :thinking:.

Reply:
@javierm what I have done for now is add a heading with a help text which explains the purpose of the map.  I think it is possibly useful to have the map because if the mapping is set up correctly in Settings, it should be a visual representation of the area the geozones relate to.

Something to think about for a future enhancement would be whether or not it is also useful to display the html image map on this page.

If the HTML map was also available on the Geozones page then maybe it would make sense to have an action button which toggled the display of each type of map on and off

There is possibly also a need for a setting which allows the admin to decide whether to display GeoJSON maps, HTML maps (or unlikely both). That way the Geojson maps could be dropped into the sidebar of Proposals etc where the HTML image map is currently displayed  or on /proposals/map